src/functionalTest/java/uk/gov/hmcts/futurehearings/hmi/functional/videohearing/VideoHearingTest.java

```diff
@@ -78,4 +78,82 @@ public void testVideoHearing() throws Exception {
                 authorizationToken,
                 null);
     }
+
+    @Test
+    public void testAmendVideoHearing() throws Exception {
+
+        log.debug("In the testSuccessfulPostVideoHearing() method");
+        final String username = String.format("abc" + new Random().nextInt(999999));
+        final String payloadForVideoHearing =
+                String.format(readFileContents("uk/gov/hmcts/futurehearings/hmi/functional/videohearing/input/POST-video-hearing-request.json"), username);
```

Comment:
Make the Directory File path as a static variable....as it is used in other tests.